src/main/resources/liquibase/workbench_changelog/21_1_0.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet author="mderamos" id="v21.1.0-1">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT count(*) FROM workbench_sidebar_category_link WHERE tool_name = 'manage_program';
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			Remove old 'Manage Program Settings' link
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			DELETE FROM workbench_sidebar_category_link
```

Comment:
Please consider the scenario  where a DM enabled the old module using the id in permissions table. 
In that case I would replace the old id by the new one
Finally delete in workbench_sidebar_category_link 
@mderamos